packages/starknet-snap/index.html

```diff
@@ -255,6 +255,16 @@ <h1>Hello, Snaps!</h1>
         <input type="submit" id="signTypedDataMessage" value="Sign Message" />
       </fieldset>
     </form>
+    <form id="signTransaction">
+      <fieldset>
+        <legend>Sign Transaction</legend>
+        <label for="transactions">Transactions</label>
+        <input type="text" id="transactions" name="transactions" /><br />
```

Comment:
add `signTransaction_` as prefix to id and name  of those input field and label (except button) , to avoid any duplication

Reply:
there is no duplications here

Reply:
yes, but just to skip the duplication if there is any new method added